.github/scripts/build_triton_wheel.py

```diff
@@ -162,7 +162,8 @@ def build_triton(
 
         # change built wheel name and version
         env["TRITON_WHEEL_NAME"] = triton_pkg_name
-        env["TRITON_WHEEL_VERSION_SUFFIX"] = version_suffix
+        if device != "xpu":
+            env["TRITON_WHEEL_VERSION_SUFFIX"] = version_suffix
```

Comment:
Avoid to duplicated git commit suffix, refer to https://github.com/triton-lang/triton/pull/4812/files#diff-eb8b42d9346d0a5d371facf21a8bfa2d16fb49e213ae7c21f03863accebe0fcfR650